### ISSUE TITLE
No accept header set when fetching context

### ISSUE THREAD

**Issue by jstcki** [rdf-ext/rdf-fetch-lite#2]
When fetching JSON-LD context, no accept header is set, which for example doesn't work at http://ld.stadt-zuerich.ch/context/flaechen.context.json

I'm not entirely sure what the correct thing to do should be (setting explicit options, pass through the options of the original request). In any case, setting `{headers:
 {accept: 'application/ld+json'}}` as `fetch` options works.

https://github.com/rdf-ext/rdf-fetch-lite/blob/9e574b83aa7d8d80c9ee41feb7141ee01de61dca/lib/patch-response.js#L31

**Comment by l00mi** [comment]
Better use the https://github.com/rdf-ext/rdf-fetch to start of. This version needs explicitly to handle the formats (as an option) which will set the Accept Header.

**Comment by jstcki** [comment]
Yes, that's what I was doing. The problem is in this library though 😅

**Comment by l00mi** [comment]
Sorry, my bad. I understand now the problem.

@bergos as the Link Header Context ist a JSON speciality I guess it is save to hard code to application/ld+json ?
